### PR TITLE
New package: dictd-1.12.1

### PR DESCRIPTION
**Pull request by CMB**
Includes its dependency, libmaa.

### CODE REVIEW

File: srcpkgs/dictd/template [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+conf_files="/etc/dict/dictd.conf /etc/dict/site.info"
+hostmakedepends="flex libtool"
+makedepends="zlib-devel libmaa-devel"
+short_desc="DICT protocol server"
+maintainer="Christopher Brannon <chris@the-brannons.com>"
+license="GPL-2"
+homepage="http://sourceforge.net/projects/dict/"
+distfiles="${SOURCEFORGE_SITE}/dict/${pkgname}-${version}.tar.gz"
+checksum=a237f6ecdc854ab10de5145ed42eaa2d9b6d51ffdc495f7daee59b05cc363656
+system_groups="dictd"
```
Comment (xtraeme) [inline srcpkgs/dictd/template]: You don't need to declare `system_groups` for the same account, `system_accounts` is enough.

File: srcpkgs/dictd/template [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+		vinstall ${FILESDIR}/colorit.conf 644 etc/dict
+		vmove usr/bin/dict
+		vmove usr/bin/colorit
+		vmove usr/bin/dictl
+		vmove usr/bin/dict_lookup
+		vmove usr/share/man/man1/dict.1
+		vmove usr/share/man/man1/colorit.1
+		vmove usr/share/man/man1/dictl.1
+		vmove usr/share/man/man1/dict_lookup.1
+	vmove usr/share/doc/dictd/examples/dict1.conf
```
Comment (xtraeme) [inline srcpkgs/dictd/template]: Please fix the indentation.

File: srcpkgs/libmaa/template [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+build_style=gnu-configure
+hostmakedepends="flex libtool"
+short_desc="Provides many low-level data structures useful for writing compilers"
+maintainer="Christopher Brannon <chris@the-brannons.com>"
+license="GPL-2"
+homepage="http://sourceforge.net/projects/dict/"
+distfiles="${SOURCEFORGE_SITE}/dict/${pkgname}-${version}.tar.gz"
+checksum=59a5a01e3a9036bd32160ec535d25b72e579824e391fea7079e9c40b0623b1c5
+
+# pre_configure() {
```
Comment (xtraeme) [inline srcpkgs/libmaa/template]: Please remove those comments if you don't need them.

Comment (CMB) [inline srcpkgs/libmaa/template]: Thank you.  All these issues should be fixed.

**Comment by xtraeme** [comment]
Please rebase and remove those redundant commits.

**Comment by CMB** [comment]
> Please rebase and remove those redundant commits.

Done.  The rebase step isn't obvious from the CONTRIBUTING.md document,
so I didn't know if it was necessary.  Maybe the clean history
should have given me a clue...  I'll send a pull request for
CONTRIBUTING.md soon, because I suspect someone else will do the same
thing I did.

**Comment by xtraeme** [comment]
Thanks :+1:

**Comment by CMB** [comment]
reopened

**Pull request closed without merging**